oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreBuilder.java

```diff
@@ -100,6 +100,20 @@
     static final int MANY_CHILDREN_THRESHOLD = Integer.getInteger(
             "oak.documentMK.manyChildren", 50);
 
+    /**
+     * The threshold value after which the document store should start (if enabled) throttling.
+     */
+    // For mongo based document store this value is threshold for the oplog replication window.
+    public static final int DEFAULT_THROTTLING_THRESHOLD = Integer.getInteger(
+            "oak.documentMK.throttlingThreshold", 2);
+
+    /**
+     * The default throttling time (in millis) when throttling is enabled. This is the time for
+     * which we block any data modification operation when system has been throttled.
+     */
+    public static final long DEFAULT_THROTTLING_TIME_MS = Long.getLong(
+            "oak.documentMK.throttlingTime", 20);
+
```

Comment:
I think these could be moved to MongoDocumentStore (and then consequently renamed to `"oak.mongo..."`) as they are now specific to mongo only